FWCore/Framework/interface/ModuleTypeResolverMakerFactory.h

```diff
@@ -0,0 +1,16 @@
+#ifndef FWCore_Framework_itnerface_ModuleTypeResolverFactory_h
+#define FWCore_Framework_itnerface_ModuleTypeResolverFactory_h
+
+#include "FWCore/PluginManager/interface/PluginFactory.h"
+
+#include <string>
+#include <vector>
+
+namespace edm {
+  class ModuleTypeResolverMaker;
+  class ParameterSet;
+
+  using ModuleTypeResolverMakerFactory = edmplugin::PluginFactory<ModuleTypeResolverMaker*(edm::ParameterSet const&)>;
```

Comment:
The `edm::ParameterSet` parameter is strictly speaking a remnant of earlier stages of the development. It is used in some of the tests, but the model with which I'd proceed with Alpaka would not use it (instead the necessary information on the variant, i.e. backend, is delivered as part of the module PSet).

I left the parameter so that the mechanism would be more general, but now I'm not sure if it would really be useful, so I could also remove it.

Reply:
Removed the `edm::ParameterSet` parameter.